avaframe/com1DFA/com1DFA.py

```diff
@@ -2654,7 +2721,7 @@ def getSimTypeList(standardCfg, simTypeList, inputSimFiles):
     entResInfo = inputSimFiles["entResInfo"]
 
     # check if set simType is a valid option
-    validSimTypesStr = "available|null|ent|entres|res"
+    validSimTypesStr = "available|null|ent|entres|res|det"
```

Comment:
This also ties into the question, is it res with det, or is det treated separately? We need to discuss this :-)